src/main/java/org/folio/rest/domain/LoanRelatedFeeFineClosedEvent.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.rest.domain;
+
+import org.folio.rest.jaxrs.model.Account;
+
+import io.vertx.core.json.JsonObject;
+
+public final class LoanRelatedFeeFineClosedEvent {
```

Comment:
An `Event` is a pub/sub entity which contains some configuration properties as well as payload that attached to the event. 

This class is the payload of the event.

Reply:
Ah, ok. This might be where the naming is a little confusing. The pub-sub `Event` is really an `EventMessage` or an `EventNotification` rather than the event itself.

As that cannot be changed because it is part of a the library (I assume) I'll resolve this as out of scope.